### PR TITLE
WebAPI: Update Method pages to modern structure (part 36)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary
Updates `### Parameters` sections.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/document/getelementsbyname/index.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ getElementsByName(name)
 
 ### Parameters
 
-- _name_ is the value of the `name` attribute of the element(s).
+- `name`
+  - : The value of the `name` attribute of the element(s).
```
Comment (teoli2003) [inline files/en-us/web/api/document/getelementsbyname/index.md:28]:
```suggestion
  - : The value of the `name` attribute of the element(s) we are looking for.
```

File: files/en-us/web/api/document/getelementsbytagname/index.md [hunk 2]
```diff
@@ -27,8 +27,9 @@ getElementsByTagName(name)
 
 ### Parameters
 
-- _name_ is a string representing the name of the elements. The special
-  string "\*" represents all elements.
+- `name`
+  - : A string representing the name of the elements. The special
+    string "\*" represents all elements.
```
Comment (teoli2003) [inline files/en-us/web/api/document/getelementsbytagname/index.md:32]:
```suggestion
    string `*` represents all elements.
```

File: files/en-us/web/api/element/requestfullscreen/index.md [hunk 3]
```diff
@@ -47,13 +47,16 @@ requestFullscreen(options)
     - `navigationUI` {{optional_inline}}
       - : Controls whether or not to show navigation UI while the element is in fullscreen mode.
         The default value is "auto", which indicates that the browser should decide what to do.
```
Comment (teoli2003) [inline files/en-us/web/api/element/requestfullscreen/index.md:49]: As we have some other changes to do:

```suggestion
        The default value is `"auto"`, which indicates that the browser should decide what to do.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/document/getelementsbytagname/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/document/getelementsbytagname/index.md]: This escape was for markdown. As `*` is special character there. But current lint configuration doesn't complain about it so removing the escape is fine.

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Pull request merged**